src/plugins/theme/boxShadow.js

```diff
@@ -7,7 +7,5 @@ module.exports = function () {
     'flat': 'none',
     sm: '0px 1px 2px rgba(0, 0, 0, 0.25)',
     md: '0px 10px 20px rgba(0, 0, 0, 0.15), 0px 6px 6px rgba(0, 0, 0, 0.2)',
-    lg: '0px 14px 28px rgba(0, 0, 0, 0.2), 0px 5px 10px rgba(0, 0, 0, 0.22)',
-    xl: '0px 19px 38px rgba(0, 0, 0, 0.25), 0px 5px 12px rgba(0, 0, 0, 0.18)',
```

Comment:
Why remove these?

Reply:
We found that we never touched the two deepest shadows.